README.md

```diff
@@ -6,7 +6,7 @@
 | :--:     | :---:               | :---:      | :---:          | :---:            | :---:                 | :---:                     |
 | [6.x][]  | **Maintenance LTS** | [Boron][]  | 2016-04-26     | 2016-10-18       | 2018-04-30            | April 2019                |
 | [8.x][]  | **Active LTS**      | [Carbon][] | 2017-05-30     | 2017-10-31       | April 2019            | December 2019<sup>1</sup> |
-| [10.x][] | **Current Release** | Dubnium    | 2018-04-24     | October 2018     | April 2020            | April 2021                |
+| [10.x][] | **Current Release** | Dubnium    | 2018-04-24     | 2018-10-30       | April 2020            | April 2021                |
```

Comment:
Should probably update **Current Releaes** at the same time

Reply:
lets do this in another commit